src/main/java/com/vistatec/ocelot/Ocelot.java

```diff
@@ -28,20 +28,9 @@
  */
 package com.vistatec.ocelot;
 
-import com.google.inject.Guice;
-import com.google.inject.Injector;
-import com.vistatec.ocelot.di.OcelotModule;
-import com.vistatec.ocelot.events.OpenFileEvent;
-import com.vistatec.ocelot.events.api.OcelotEventQueue;
-import com.vistatec.ocelot.its.view.ProvenanceProfileView;
-import com.vistatec.ocelot.plugins.PluginManagerView;
-import com.vistatec.ocelot.rules.FilterView;
-import com.vistatec.ocelot.rules.QuickAddView;
-import com.vistatec.ocelot.segment.model.OcelotSegment;
-import com.vistatec.ocelot.segment.view.SegmentAttributeView;
-import com.vistatec.ocelot.segment.view.SegmentView;
-import com.vistatec.ocelot.ui.ODialogPanel;
-
+//<<<<<<< HEAD
+//=======
+//>>>>>>> marta_branch
```

Comment:
There's a bunch of places like this where the merge conflict was just commented out.  Please remove these instead -- otherwise the code will become quite confusing over time.